src/d3a/d3a_core/redis_connections/aggregator_connection.py

```diff
@@ -30,8 +23,15 @@ def __init__(self, redis_db):
         self.lock = Lock()
         self.grid_buffer = {}
 
-    def set_aggregator_device_mapping(self, aggregator_device_mapping):
-        self.aggregator_device_mapping = aggregator_device_mapping
+    @property
+    def _aggregator_info(self):
+        """
+        to be implemented
+        """
+        return {}
```

Comment:
Didnt read the discussion closely, will remove this as it is not needed any more.